### ISSUE TITLE
Improve Analysis UI

### ISSUE THREAD

**Issue by laveesh** [WFP-VAM/prism-app#98]
After #92 has been merged

It is the extension of the #86 
This is in the continuation to [New Analysis UI Footer](https://github.com/WFP-VAM/prism-frontend/issues/86#issuecomment-657021260)
The new Analysis UI
![New Analysis UI Footer](https://user-images.githubusercontent.com/31915056/87220750-52c59000-c384-11ea-9cee-baa562b5e9e2.png)

**TODO**
- [ ] Add new footer UI
- [x] Improve clear analysis  functionality
- [x] Improve Download functionality
- [ ] Refactor the analysis layer
- [ ] Improve the implementation of tooltip on the map

**Comment by SimplyBallistic** [comment]
Finding a way to refactor the analysis layer so it either extends upon an impact layer (quite similar in purpose) or integrates as a `LayerType` would be great, however this will not be straightforward as the layer system is built primarily around JSON.

**Comment by wadhwamatic** [comment]
No longer relevant. Closing
